### PR TITLE
block worlds zoom + eye wheel

### PR DESCRIPTION
**Pull request by fokkonaut**

### CODE REVIEW

**Comment by ZombieToad** [comment]
i dont think this should be added. how long is blockworlds gonna be around? its not opensource and doesnt have public releases. also vali sucks

**Comment by fokkonaut** [comment]
xD true but ddnet players are "forced" to download ath to zooz there. if bw is down we can easily remove this little code again.

**Comment by def-** [comment]
Why not just add it to the IsDDNet()? If it is derived from DDNet and everything that's allowed on DDNet is allowed on BlockWorlds, then that's the way to go.

File: src/engine/shared/serverbrowser.cpp [hunk 1]
```diff
@@ -47,7 +47,9 @@ bool IsDDRace(const CServerInfo *pInfo)
 bool IsDDNet(const CServerInfo *pInfo)
 {
 	return str_find_nocase(pInfo->m_aGameType, "ddracenet")
-	    || str_find_nocase(pInfo->m_aGameType, "ddnet");
+	    || str_find_nocase(pInfo->m_aGameType, "ddnet")
+	    || ((str_comp_nocase_num(pInfo->m_aGameType, "bw", 2) == 0 && str_length(pInfo->m_aGameType) > 3 && pInfo->m_aGameType[2] == ' ' && pInfo->m_aGameType[3] == ' ')
```
Comment (def-) [inline src/engine/shared/serverbrowser.cpp]: What is this line doing? Why not str_comp_nocase("bw  ")?

Comment (fokkonaut) [reply src/engine/shared/serverbrowser.cpp]: ath made it that way

**Pull request merged**